.github/workflows/misc.yml

```diff
@@ -24,10 +24,8 @@ jobs:
       - name: Environment Information
         run: npx envinfo
       - name: Build
-        run: NODE=$(command -v node) make doc-only
+        run: NODE=$(command -v node) make test-doc
```

Comment:
This will run the markdown linter as well, which is already running in the linter workflow.

Reply:
Maybe it's still better than not running doctool suite at all on GH Actions? I can't find an easy way to tell make to ignore prerequisites.

Reply:
Addressed, I've added a `test-doc-ci` target which doesn't run the linters.